Microsoft.Dynamics365.UIAutomation.Browser/Logs/TracingService.cs

```diff
@@ -0,0 +1,129 @@
+﻿using System;
```

Comment:
This two classes can be useful also in project, to easily create more expressive Traces.